### PR TITLE
Add support for VLX addresses

### LINKED ISSUE #139: Add support for VLX addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

We will likely be reviewing PRs and paying out bounties more or less on a weekly basis. No need to ping us to check your PR, we will definitely get to it.

Thanks for your help!

### PR DESCRIPTION
**Pull request by Echolon166**
## Issue number

Closes #139 

## Description

Add support for VLX addresses

## Reference
- https://github.com/velas/GoVelas/blob/705ab73a93b150e9ee3d13b5b0846c8ee97453fb/crypto/wallet.go#L10
- https://github.com/velas/NVelas/blob/73b5c1f752d7c24149cc037960524829f037814d/docs/crypto/wallet.md

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by makoto** [comment]
@Echolon166  Please resolve conflicts

**Comment by makoto** [comment]
@Echolon166  Please resolve conflicts

**Comment by Echolon166** [comment]
@makoto Resolved.

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ This library currently supports the following cryptocurrencies and address forma
  - SYS (base58check P2PKH and P2SH, and bech32 segwit)
  - TRX (base58check)
  - VET (checksummed-hex)
+ - VLX (base58)
```
Comment (makoto) [inline README.md:78]: Please change this to base58, no check (and revolve conflict again)

Comment (Echolon166) [reply README.md]: Done.

**Comment by makoto** [comment]
Sorry but can you resolve conflict again?

**Pull request merged**